client/pyre.py

```diff
@@ -144,8 +144,8 @@ def writable_path(path: str) -> str:
         action="append",
         default=[],
         type=readable_directory,
-        help="Additional directories with modules and stubs "
-        "to include in type environment",
+        help="adds an additional directory of modules and stubs to include in type"
+        " environment each time it is passed",
```

Comment:
I am essentially bikeshedding here, but I'd like to keep this consistent with the surrounding options.

I would prefer something like "Add an additional directory of modules and stubs to include in type environment":
* Starts with uppercase
* Uses imperative form rather than the present tense (I know, "Preserves the value of the PYTHONPATH..." just below is wrong)
* Avoids redundancy with the "each time it is passed" part

Reply:
Sure thing.